server/cp/ingest/parser/ap.py

```diff
@@ -851,6 +859,10 @@ def _parse_ap_products(self, data, item):
             for p in products
         ])
 
+    def categorisation_mapping(self, in_item, item):
```

Comment:
Is this function used somewhere?

Reply:
it's called by the parent class doing some mapping which should not be used by cp

Reply:
okay.